ui/src/app/utils/leo-runtime-initializer.tsx

```diff
@@ -83,6 +83,7 @@ export interface LeoRuntimeInitializerOptions {
   maxResumeCount?: number;
   maxServerErrorCount?: number;
   targetRuntime?: Runtime;
+  resolutionCond?: (Runtime) => boolean;
```

Comment:
What does this name mean?

Reply:
it's a function that must return true in order for the polling to "resolve" and exit. I added it so we can resolve on conditions other than just "== Running"

Reply:
hm, hard to name something like that. Perhaps something like `stopPollingPredicate`?